src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -491,7 +492,7 @@ public static boolean postOMFBatch(String payload) {
     String postHeaderType = "&messagetype=type";
 
     postHeaderType = "&messagetype=data";
-    String responseFilename = "/usr/response.txt";
+    String responseFilename = "/usr/response.json";

```

Comment:
Why is this duplicated. Also the capitalization is inconsistent. 

The response directory and filename should probably be class finals.